app/views/posts/show.html.erb

```diff
@@ -1,3 +1,34 @@
-<h1>This is the place for all posts.</h1>
+<section class="showP">
+
+  <div class="postL">
+    <h3>
+      <%= @post.title %>
+    </h3>
+    <p>
+      <%= @post.text %>
+    </p>
+    <span class="like">
+      Likes: <%= @post.likes.count || 0 %>
+    </span>
+    <span class="comment">
+      Comments: <%= @post.comments.count || 0 %>
+    </span>
+  </div>
+
+  <ul class="commentP">
+    <% if @post.comments.length == 0 %>
+      <p>
+        No comments yet
+      </p>
+    <% else %>
+      <% @post.comments.each do |comment| %>
+        <li>
+          <%= comment.users.name %>:
+          <%= comment.text %>
+        </li>
+      <% end %>
```

Comment:
- [ ] There is a bug 🐛 on your posts show page as you can see in the image attached below. I narrowed it down to the relationship between your comment and users on line 26. Instead of `comment.users.name`, you should have `comment.author.name`. I believe this should solve the issue 👍 
![image](https://user-images.githubusercontent.com/46839548/198272193-f0a8c3ce-011f-40e6-b9f0-64da08b51c95.png)